src/Microsoft.ML.Data/Transforms/NormalizeColumn.cs

```diff
@@ -228,10 +228,12 @@ public sealed class SupervisedBinArguments : BinArgumentsBase
         public const string SupervisedBinNormalizerShortName = "SupBin";
 
         /// <summary>
-        /// A helper method to create MinMaxNormalizer transform for public facing API.
+        /// Convenience method for instantiating <see cref="NormalizeTransform"/> for Min-Max normalization.
```

Comment:
>Convenience method for instantiating [](start = 12, length = 36)

Instantiates  - I don't think we need to stress the word convenience.

Reply:
IMO, we have to because there are other constructors/methods as well which instantiate the object but are not that handy.

Reply:
IMO, we have to because there are other constructors/methods as well which instantiate the object but are not that handy.

---
In reply to: [207688744](https://github.com/dotnet/machinelearning/pull/646#discussion_r207688744) [](ancestors = 207688744)

Reply:
Default language for C# constructor documentation is: "Initializes a new instance of <see cref="">
You can add another line that conveys what it does. 

---
In reply to: [207691640](https://github.com/dotnet/machinelearning/pull/646#discussion_r207691640) [](ancestors = 207691640,207688744)